APMrover2/control_modes.cpp

```diff
@@ -117,3 +117,14 @@ void Rover::read_trim_switch()
     }
 }
 
+bool Rover::is_moving()
```

Comment:
perhaps it would be more accurate to call this motor_active() instead of is_moving() ?